Cannot cmake razercfg-0.28

Running `cmake .` yields the following error:

```
CMake Error at CMakeLists.txt:53 (install):
  install FILES given no DESTINATION!
```

The indicated line of CMakeLists.txt (53-57):

```
 install(FILES ${CMAKE_BINARY_DIR}/razerd.service
       DESTINATION ${SYSTEMD_UNIT_DIR}
       PERMISSIONS OWNER_READ OWNER_WRITE
               GROUP_READ
               WORLD_READ)
```

```
$ cmake --version
cmake version 2.8.12.2
```

```
$ uname -a
Linux jeffrey-desktop 3.13.0-30-generic #55-Ubuntu SMP Fri Jul 4 21:40:53 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux
```

```
$ cat /etc/*-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=14.04
DISTRIB_CODENAME=trusty
DISTRIB_DESCRIPTION="Ubuntu 14.04 LTS"
NAME="Ubuntu"
VERSION="14.04, Trusty Tahr"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 14.04 LTS"
VERSION_ID="14.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -27,7 +27,8 @@ add_subdirectory(librazer)
 configure_file("udev.rules.template" "udev.rules")
 
 execute_process(COMMAND pkg-config --variable=udevdir udev
-  OUTPUT_VARIABLE UDEV_DIR OUTPUT_STRIP_TRAILING_WHITESPACE)
+  OUTPUT_VARIABLE UDEV_DIR
+  OUTPUT_STRIP_TRAILING_WHITESPACE)
 
 install(FILES ${CMAKE_BINARY_DIR}/udev.rules
 	DESTINATION ${UDEV_DIR}/rules.d
@@ -48,13 +49,17 @@ install(FILES ${CMAKE_BINARY_DIR}/pm-hook.sh
 configure_file("razerd.service.template" "razerd.service")
 
 execute_process(COMMAND pkg-config --variable=systemdsystemunitdir systemd
-  OUTPUT_VARIABLE SYSTEMD_UNIT_DIR OUTPUT_STRIP_TRAILING_WHITESPACE)
+  OUTPUT_VARIABLE SYSTEMD_UNIT_DIR
+  ERROR_QUIET
+  OUTPUT_STRIP_TRAILING_WHITESPACE)
 
-install(FILES ${CMAKE_BINARY_DIR}/razerd.service
-	DESTINATION ${SYSTEMD_UNIT_DIR}
-	PERMISSIONS OWNER_READ OWNER_WRITE
-		    GROUP_READ
-		    WORLD_READ)
+if(SYSTEMD_UNIT_DIR)
+	install(FILES ${CMAKE_BINARY_DIR}/razerd.service
+		DESTINATION ${SYSTEMD_UNIT_DIR}
+		PERMISSIONS OWNER_READ OWNER_WRITE
+					GROUP_READ
+					WORLD_READ)
+endif(SYSTEMD_UNIT_DIR)
 
 install_exec_cmd("udevadm control --reload-rules"
 		 "Please reload udev manually or reboot the system")

